src/axes/categoryAxis.ts

```diff
@@ -13,8 +20,8 @@ namespace Plottable.Axes {
      */
     private get _wrapper() {
       const wrapper = new SVGTypewriter.Wrappers.Wrapper();
-      if (this._tickLabelMaxWidth != null) {
-        wrapper.maxLines(1);
+      if (this._tickLabelMaxLines != null) {
+        wrapper.maxLines(this._tickLabelMaxLines)
```

Comment:
This is the same code as the one in plottable.js; I committed the built file as well